webapp/src/webapp/auth/local/login.cljs

```diff
@@ -0,0 +1,51 @@
+(ns webapp.auth.local.login
+  (:require
+    ["@radix-ui/themes" :refer [Flex Card Heading
+                                Link Box Text]]
+    [webapp.components.button :as button]
+    [webapp.components.forms :as forms]
+    [reagent.core :as r]
+    [re-frame.core :as re-frame]))
+
+(defn- form []
+  (let [email (r/atom "")
+        password (r/atom "")
+        loading (r/atom false)]
+    (fn []
+      [:> Box
+       [:> Flex {:direction "column"}
+        [forms/input {:label "Email"
+                      :value @email
+                      :type "email"
+                      :on-change #(reset! email (-> % .-target .-value))}]
+        [forms/input {:label "Password"
+                      :value @password
+                      :type "password"
+                      :on-change #(reset! password (-> % .-target .-value))}]
+        [button/tailwind-primary
```

Comment:
here you can use the button of Radix default color indigo.
Example:
`[:> Button {:color "indigo"
                :size "2"
                :variant "solid"
                :radius "medium"}
     "Login"]`